geotrek/common/templates/common/publishable_completeness_fragment.html

```diff
@@ -9,7 +9,11 @@
 {% else %}
     {% if not object.is_publishable %}
         <div class="alert alert-danger"><strong>{% trans "Object is published but" %}</strong>
-        {% if not object.is_complete %}{% trans "looks incomplete;" %}{% endif %}
+        {% if not object.is_complete %}
+            {% trans "looks incomplete;" %}
+            {% trans "these fields should be completed:" %}<br/>
+            {{ completeness_fields|join:', ' }}
```

Comment:
I think you should used translated names for field, not internal names

Reply:
Now the view send verbose name instead of internal names.